01/assets/js/app.js

```diff
@@ -1,4 +1,59 @@
 const a = '4.2';
 const b = 9;
 
-console.log(a, b);
\ No newline at end of file
+console.log(a, b);
+
+console.log('a: ', typeof(a))
+console.log('b: ', typeof(b))
+
+const suma = Number(a) + Number(b);
+const roznica = Number(a) - Number(b);
+const iloczyn = Number(a) * Number(b);
+const iloraz = Number(a) / Number(b);
+const modulo = Number(a) % Number(b);
```

Comment:
Staraj się używać angielskich słów - jeśli masz problem ze słownictwem to proponuję Google Translator (sam korzystam :D)